### DISCUSSION TITLE
The default value is incorrect when clearing the number format settings

### DISCUSSION THREAD

**Discussion by pcsuccess** [microsoft/vscode-discussions#204] [Extension Development QnA] [answered]
When the content format of the extension setting item is a number, why does it default to the first digit of the previously set number after clearing (for example, the default value after 987 is cleared is 9)? 
Is there a way to set the default value after clearing? 
Have you encountered this situation? We look forward to your reply.
![image](https://user-images.githubusercontent.com/39364402/198551976-9bb4da78-1500-440f-a7f3-2d5c2e2f454e.png)
![image](https://user-images.githubusercontent.com/39364402/198555032-418c9907-c4e9-491f-9456-2a66c0ceb61b.png)

**Comment by gjsjohnmurray** [comment, marked as answer]
The context menu that appears when you click the gearwheel includes a "Reset Setting" option.

But notice the "(Also modified in User)" message. You are probably viewing the Workspace tab of Settings Editor. Switch to the User tab and see what value you have set it to there.
